src/client/driver/command-executors/ensure-element.js

```diff
@@ -0,0 +1,37 @@
+import hammerhead from '../deps/hammerhead';
+import testCafeCore from '../deps/testcafe-core';
+
+
+var nativeMethods = hammerhead.nativeMethods;
+
+var waitFor       = testCafeCore.waitFor;
+var positionUtils = testCafeCore.positionUtils;
+
+
+const CHECK_ELEMENT_DELAY = 200;
+
+
+function ensureElementExists (selector, timeout, createError) {
```

Comment:
I guess we can move `ensureElementEditable`, `ensureTextAreaElement`, etc. to this file as well